manager/src/main/java/com/bakdata/quick/manager/k8s/resource/QuickResources.java

```diff
@@ -50,4 +50,11 @@ enum ResourcePrefix {
             this.prefix = deploymentPrefix;
         }
     }
+
+    /**
+     * A method for retrieving the name of resources (without prefix).
+     * @return The name that a user has chosen for a specific type of resources (gateway, mirror, app)
```

Comment:
```suggestion
     * @return The name that a user has chosen for a specific type of resources
```
Even though it's just a comment, don't assume who implements this interface